serverless-hello-world/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "http-handler",
+  "version": "0.1.0",
+  "description": "A function which responds to HTTP requests",
+  "main": "index.js",
+  "scripts": {
+    "test": "node test/test.js",
+    "start": "faas-js-runtime ./index.js",
+    "debug": "nodemon --inspect ./node_modules/faas-js-runtime/bin/cli.js ./index.js"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "Apache-2.0",
```

Comment:
Looks like we are defining the license as Apache 2, but there is no license file in the PR?